app/miragevpn_server_notun.ml

```diff
@@ -213,10 +211,12 @@ let callback t fd =
 
 let connect config =
   let open Lwt.Infix in
+  let connections = Hashtbl.create 7 in
```

Comment:
hmm, I doubt the subnet size is the expected amount of connected clients (and e.g. if you specify a /16 and only have 1 client, it sounds rather wasteful) ;) we can modify the initial size when we have good arguments or use cases. but I'd like to avoid doing it depending on premature optimization.